regions/io/ds9/tests/test_ds9_language.py

```diff
@@ -196,3 +203,10 @@ def test_ds9_color_override_global():
     assert regions[0].meta['color'] == 'green'
     assert regions[1].meta['color'] == 'orange'
     assert regions[2].meta['color'] == 'blue'
+
+def test_issue134_regression():
+    regstr = 'galactic; circle(+0:14:26.064,+0:00:45.206,30.400")'
```

Comment:
@joleroi this is a little redundant with your test, but it is a different case b/c it's `;`-separated, so I think it's reasonable to include.